values.yaml

```diff
@@ -1,402 +1,134 @@
-# storageClassName -- Sets the storage class for all Coder services and user
-# environments. By default, the storageClassName is unspecified, and thus the
-# default StorageClass is used. If storageClassName is not specified and a
-# default StorageClass does not exist, then the deployment will fail. The
-# storageClass MUST support the ReadWriteOnce access mode.
-storageClassName: ""
-
-# serviceType -- See the following for the different serviceType options and
-# their use:
-# https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
-serviceType: "ClusterIP"
-
-# podSecurityPolicyName -- The name of the pod security policy to apply to all
-# Coder services and user environments. The optional ingress has its own field
-# for pod security policy as well.
-podSecurityPolicyName: ""
-
-# clusterDomainSuffix -- If you've set a custom default domain for your
-# cluster, you may need to remove or change this DNS suffix for service
-# resolution to work correctly.
-clusterDomainSuffix: ".svc.cluster.local"
```

Comment:
Can't go if we support bundled postgres. Multiple customers have custom suffixes.